api-tester/src/main/java/com/revenuecat/apitester/java/DeprecatedPurchasesAPI.java

```diff
@@ -15,7 +15,7 @@ static void check(final Purchases purchases,
                       final SkuDetails skuDetails,
                       final StoreProduct storeProduct,
                       final Package packageToPurchase,
-                      final UpgradeInfo upgradeInfo) {
+                      final ProductChangeInfo productChangeInfo) {
```

Comment:
Hmm since this would be a breaking change, ideally we could deprecate UpgradeInfo and the API that uses it. We can convert from UpgradeInfo to ProductChangeInfo internally. That way  the migration is easier for devs. I'm afraid of the number of methods we will have with this, but we should try to avoid breaking changes wherever possible

Reply:
this is my bad... i told josh we could just go ahead and remove it... we removed/renamed the two properties within it, so it felt unhelpful to keep the class itself around. but you're right that it's probably the right thing to do.. 

i will say, i think the way this is passed around will change based on the [isPersonalizedPrice](https://github.com/RevenueCat/purchases-android/pull/796) PR. might be worth waiting for that one to finish before we decide how to best deprecate this?

Reply:
Aye aye, I can wait and redo 😇